src/vaadin-avatar.html

```diff
@@ -14,6 +14,7 @@
     <style>
       :host {
         display: inline-block;
+        flex: none;
```

Comment:
Hm, does this `display: inline-block` rule have any meaning as a flex child - could it be removed?

Reply:
Not sure if we could remove it when Avatar is used inside Avatar Group. But Avatar can be used standalone as well, and then it needs to be able to define a size for itself – `inline` won’t work.